deployment/helm/values.yaml

```diff
@@ -35,13 +35,10 @@ exposedServiceType: NodePort # ClusterIP | NodePort | LoadBalancer
 # environs is a list of *additional* environs to add to the Alert's ConfigMap
 # The follwing environs are already set in the ConfigMap
 # at /templates/alert-environ-resources.yaml (go there to set them):
-#   - PUBLIC_HUB_WEBSERVER_HOST
 #   - HUB_WEBAPP_HOST
 #   - HUB_CFSSL_HOST
-#   - PUBLIC_HUB_WEBSERVER_PORT
 #   - ALERT_SERVER_PORT
-#   - PUBLIC_HUB_WEBSERVER_PORT
-# Format: array of strings delimited by commas; each string has the format "KEY:VALUE" e.g. ["PUBLIC_HUB_WEBSERVER_HOST:localhost", "PUBLIC_HUB_WEBSERVER_PORT:443"]
+# Format: array of strings delimited by commas; each string has the format "KEY:VALUE" e.g. ["HUB_WEBAPP_HOST:localhost", "ALERT_SERVER_PORT:443"]
```

Comment:
Does this replacement suffice? I'm not familiar with helm.

Reply:
This appears to be correct.  More testing with Helm is needed.  The files have been added to Alert but more testing is needed on our part.